dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/jobService/data/PutJobData.kt

```diff
@@ -117,6 +125,15 @@ data class PutJobData(
         val parent = item.second.parent
         val paths = item.second.toFile().walk(FileWalkDirection.TOP_DOWN)
                     .filter { !(it.isDirectory && Files.list(it.toPath()).use { f -> f.findAny().isPresent }) }
+                    .filter {
+                     try {
```

Comment:
I would pull this logic out into a function to clean up the function call chain.

Reply:
That's a good idea I'll get that swapped around.